File: app/src/main/java/com/qweex/quite/ImageFragment.java
package com.qweex.quite;

import android.graphics.Bitmap;
import android.graphics.BitmapFactory;
import android.view.View;
import android.widget.ImageView;

public class ImageFragment extends FragmentBase {
    static final String[] filesHandled = new String[] {".jpg", ".jpeg", ".png"};
    ImageView v;

    @Override
    public View initView() {
        if(v==null) {
            v = new ImageView(context);
            v.setId(R.id.content_view);
        }
        Bitmap myBitmap = BitmapFactory.decodeFile(currentPath);
        v.setImageBitmap(myBitmap);
        return v;
    }
}

File: app/src/main/java/com/qweex/quite/ImageFragment.kt
package com.qweex.quite

import android.graphics.BitmapFactory
import android.view.View
import android.widget.ImageView

class ImageFragment : FragmentBase() {
    internal var v: ImageView? = null

    public override fun initView(): View {
        if (v == null) {
            type = "Image"
            v = ImageView(context)
            v!!.id = R.id.content_view
            v!!.setOnLongClickListener((activity as MainActivity).showOptions)
        }
        val myBitmap = BitmapFactory.decodeFile(currentPath)
        v!!.setImageBitmap(myBitmap)
        return v as ImageView
    }

    companion object {
        internal val filesHandled = arrayOf(".jpg", ".jpeg", ".png") as Array<String?>
    }
}
